test/ui/BandCompTest.js

```diff
@@ -0,0 +1,131 @@
+const webdriver = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
+const chromedriver = require('chromedriver');
+const By = webdriver.By
+chrome.setDefaultService(new chrome.ServiceBuilder(chromedriver.path).build());
+var map = webdriver.promise.map;
+var chai = require('chai');
+var should = require('chai').should();
+var axios = require('axios');
+var MockAdapter = require('axios-mock-adapter');
+const JobService = require('../../app/service/JobService');
+const expect = chai.expect;
+
+describe('Page band-comp should include competency and competency information', () => {
+    it('should return band competencies', async () => {
+
+        var driver = await new webdriver.Builder()
+            .setChromeOptions(new chrome.Options().headless())
+            .withCapabilities(webdriver.Capabilities.chrome())
+            .build();
+
+        //launch job-roles page
+        driver.get('http://localhost:3000/job-roles');
+
+        //find band name link and click it
+
+        let bandName = await driver.findElement(By.id("compByBandID1")).getText().then(function (value) {
```

Comment:
do you need the .then on these calls? I thought it would have worked with just the .getText calls

Reply:
I  used then. because later I need use variable bandName to make assertion, without then. test doesn't pass.